corehq/apps/ota/models.py

```diff
@@ -199,7 +199,7 @@ class DeviceLogRequest(models.Model):
     """
     domain = models.CharField(max_length=255)
     username = models.CharField(max_length=255, help_text="raw_username, no @domain.commcarehq.org")
-    device_id = models.CharField(max_length=255)
+    device_id = models.CharField(max_length=255, null=True, default='')
```

Comment:
why default to empty string?

Reply:
[_should_force_log_submission](https://github.com/dimagi/commcare-hq/blob/8f219673aa01a4083f1ecc818523b49cf4239b19/corehq/apps/ota/views.py#L364) defaults to blank string, so I did the same here